ci-operator/step-registry/openshift/unify-ci/cucushift/upgrade/openshift-unify-ci-cucushift-upgrade-commands.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
+  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
+  exit 1
+fi
+
+TO_VERSION="${RELEASE_IMAGE_LATEST}"
+# TO_VERSION="4.7.24"
+# CHANNEL="stable"
+# FORCE="true"
+
+step=20 # second
+
+# HACK: HyperShift clusters use their own profile type, but the cluster type
+# underneath is actually AWS and the type identifier is derived from the profile
+# type. For now, just treat the `hypershift` type the same as `aws` until
+# there's a clean way to decouple the notion of a cluster provider and the
+# platform type.
+#
+# See also: https://issues.redhat.com/browse/DPTP-1988
+if [[ "${CLUSTER_TYPE}" == "hypershift" ]]; then
+    export CLUSTER_TYPE="aws"
+    echo "Overriding 'hypershift' cluster type to be 'aws'"
+fi
+
+# For disconnected or otherwise unreachable environments, we want to
+# have steps use an HTTP(S) proxy to reach the API server. This proxy
+# configuration file should export HTTP_PROXY, HTTPS_PROXY, and NO_PROXY
+# environment variables, as well as their lowercase equivalents (note
+# that libcurl doesn't recognize the uppercase variables).
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    # shellcheck disable=SC1090
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+export OPENSHIFT_ENV_OCP4_ADMIN_CREDS_SPEC=${KUBECONFIG}
+export KUBECONFIG=${KUBECONFIG}
```

Comment:
`KUBECONFIG` is exported by default, https://docs.ci.openshift.org/docs/architecture/step-registry/#available-environment-variables